hot-desk/src/main/resources/application.yml

```diff
@@ -6,7 +6,7 @@ datasource:
   host: '${dockerdbhost:localhost}:3306'
   url: 'jdbc:mariadb://${datasource.host}/${datasource.schema}'
   username: 'root'
-  password: ''
+  password: '09102003'
```

Comment:
Please do not add your credentials to configs

Reply:
sorry I forgot to remove it